src/modelinstance.hpp

```diff
@@ -452,9 +452,10 @@ class ModelInstance {
 
     /**
          * @brief Unloads model version
-         *
+         * @param permanent defined if the unload operation should be permenent and should change instance stated after it is completed
```

Comment:
->         * @param change status decides if the model status code should be change during unload